src/main/scala/io/sdkman/changelogs/java/AdoptOpenJdkMigrations.scala

```diff
@@ -997,4 +997,78 @@ class AdoptOpenJdkMigrations {
   )
   def migrate0039(implicit db: MongoDatabase) =
     setCandidateDefault("java", "11.0.9.hs-adpt")
+
+  @ChangeSet(
+    order = "0040",
+    id = "0040-remove_adoptopenjdk-mac-j9_11_0_9",
+    author = "eddumelendez"
+  )
+  def migrate0040(implicit db: MongoDatabase) =
+    Seq(
+      MacOSX
+    ).foreach { platform =>
+      removeVersion("java", "11.0.9.j9-adpt", platform)
```

Comment:
You could simply reduce this to a one line removal without wrapping platform in a sequence.